cmd/gds/main.go

```diff
@@ -52,8 +52,8 @@ func main() {
 				EnvVars: []string{"TRISA_DIRECTORY_PROFILE", "GDS_PROFILE"},
 			},
 			&cli.StringFlag{
-				Name:    "endpoint",
-				Aliases: []string{"e"},
+				Name:    "directory-endpoint",
+				Aliases: []string{"d"},
```

Comment:
Could we also keep the `e` endpoint, e.g. `Aliases: []string{"d", "e"}`? It's so ingrained that I use -e for endpoint that it will probably trip me up. Also I've noticed that there is no `GlobalString` in urfave v2 anymore, which means that if an flag conflicts with a subcommand flag, e.g. `gds -d foo cmd -d bar` then "bar" will be the value taking. `-d` is commonly used for database, directory, etc.